### PR TITLE
feat(ecs): Add artifact support, operation mappings to reference page

### PR DESCRIPTION
**Pull request by allisaurus**
Artifact support added in 1.15.0:  
https://github.com/spinnaker/clouddriver/pull/3825

### CODE REVIEW

**Comment by allisaurus** [comment]
CC: @clareliguori

File: reference/providers/ecs.md [hunk 1]
```diff
@@ -7,13 +7,41 @@ sidebar:
 
 {% include toc %}
 
-## Resource Mapping
+## Resource mapping
 
 ### Server Group
```
Comment (clareliguori) [inline reference/providers/ecs.md]: Probably someone like @dorbin can help us with correct casing here.  This change has "Resource mapping" and "Operation mapping", but "Server Group" (capital G)

File: reference/providers/ecs.md [hunk 2]
```diff
@@ -7,13 +7,41 @@ sidebar:
...
 A Spinnaker **server group** maps to an [Amazon ECS service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts).
 
+There are two ways to specify server group settings for ECS:
+
+* __Inputs (default)__: 
+  Specify the container image, resource limits, and other settings needed to create an [ECS task definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) and [service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_concepts) directly in the server group. This option supports deploying one container per Amazon ECS task (equivalent to a Spinnaker "instance").
```
Comment (clareliguori) [inline reference/providers/ecs.md]: Double check on ECS -> Amazon ECS everywhere, to match our naming in the official AWS docs

File: reference/providers/ecs.md [hunk 3]
```diff
@@ -7,13 +7,41 @@ sidebar:
...
 A Spinnaker **instance** maps to an Amazon ECS [task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_run_task.html).  Amazon ECS services manage tasks to ensure desired capacity is reached.
 
 ### Cluster
 An [Amazon ECS cluster](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_clusters.html) does not map to any core Spinnaker concept.  You can choose what cluster you deploy to in your deploy stage parameters.
+
+
+## Operation mapping
+
+### Deploy
+Deploys a new Amazon ECS Service to the specified server group(s). For load balanced services, the previous service will be drained after the new service is considered healthy.
```
Comment (clareliguori) [inline reference/providers/ecs.md]: Should service be upper case or lower case?

File: reference/providers/ecs.md [hunk 4]
```diff
@@ -7,13 +7,41 @@ sidebar:
...
+Deployments to Amazon ECS take two actions in regards to Amazon ECS resources:
+
+1. Register a new ECS task definition that contains the specified container(s), their image(s), and other information needed for your application to run.
+2. Create a new ECS service that runs the registered task definition with the desired instance (task) count, load balancer, and scaling policies. 
+
+### Destroy
+Scales the desired count for the ECS service down to 0, then deletes the service.
+
+### Disable
+Scales the desired count for the ECS service down to 0, so no instances are running.
```
Comment (clareliguori) [inline reference/providers/ecs.md]: Should we continue doing "instances (tasks)" here and below in Resize?

File: reference/providers/ecs.md (hunk 1 above)
Comment (dorbin) [reply reference/providers/ecs.md]: Sadly, we've got inconsistent capitalization of "server group" on the site. I don't see it as a proper noun, and we're trying to keep our headings to sentence case. So I think it should be "Server group"

Comment (allisaurus) [reply reference/providers/ecs.md]: In [several](https://www.spinnaker.io/reference/providers/cf/) [other](https://www.spinnaker.io/reference/providers/kubernetes/) [reference](https://www.spinnaker.io/reference/providers/appengine/) pages, section headers (e.g., "Resource mapping") are cased as above. "Server Group" is more inconsistent across these same pages, however is either always or mostly capitalized on both words. 

I'll change "Server Group" to always be upper case, but defer to guidance from @dorbin  or others.

File: reference/providers/ecs.md (hunk 3 above)
Comment (allisaurus) [reply reference/providers/ecs.md]: for consistency, ECS resource types seem to be always lower case; will change.

**Review by clareliguori — APPROVED** [review]

**Comment by clareliguori** [comment]
@dorbin Now that 1.15 is out, any feedback before getting this merged?

**Comment by dorbin** [comment]
Sorry, Clare, et al, I was out at a summit all last week.

**Pull request merged**